components/SliderPlusDescription.vue

```diff
@@ -0,0 +1,52 @@
+
+<script setup>
+import { defineProps } from 'vue';
+
+  const props = defineProps({
+    slides: {
+      type: Array,
+      required: true,
+    },
+  })
+
+  const { slides } = props
+
+
+</script>
+
+
+
+<template>
+  <v-carousel>
+    <v-carousel-item
+      v-for="(slide, index) in slides"
+      :key="index"
+      :src="slide.src"
+      :alt="slide.alt"
+      :height = "370"
+      cover
+    >
+      <div class="caption">
+        <h2>{{ slide.caption.header }}</h2>
+        <p>{{ slide.caption.intro }}</p>
+      </div>
+    </v-carousel-item>
+  </v-carousel>
+</template>
+
+
+<style>
+.caption {
+  position: absolute;
+  bottom: 20px;
+  left: 50%;
```

Comment:
@AgnieszkaIwan you didn't set a transform: translateX value for this left: 50%, so it probably won't be fully  centered.

Reply:
now should be ok